csympy/lib/csympy_wrapper.pyx.in

```diff
@@ -341,28 +362,50 @@ cdef class Integer(Number):
         import sympy
         return sympy.Integer(deref(self.thisptr).__str__().decode("utf-8"))
 
+    def _sage_(self):
+        IF HAVE_SAGE == True:
```

Comment:
Ah, I forget about that! Hm, I don't see any other way, unless you can `cimport` in Cython as a try.